src/StatusQ/Popups/qmldir

```diff
@@ -8,6 +8,7 @@ StatusMenuHeadline 0.1 StatusMenuHeadline.qml
 StatusModal 0.1 StatusModal.qml
 StatusSearchPopup 0.1 StatusSearchPopup.qml
 StatusModalDivider 0.1 StatusModalDivider.qml
+StatusModalStatusBar 0.1 StatusModalStatusBar.qml
```

Comment:
is it dedicated to modal? Should it be StatusBanner?

Reply:
Certainly, it is not strictly necessary to be part of a modal. It fits with parent width so we could move it maybe to **StatusQ.Controls**. Is it a better location? 

It is a general approximation as to be more generalist it should incorporate some text length checks but it is something we can improve in the future.

It sounds good **StatusBanner**. I’ll change the component name to that.